packages/bff/src/config.ts

```diff
@@ -1,27 +1,59 @@
+import { FastifySessionOptions } from '@fastify/session';
+import 'dotenv/config';
 import z from 'zod';
 
 const envVariables = z.object({
   // todo: rather use version here instead of git_sha
   GIT_SHA: z.string().default('v6.1.5'),
-  DB_CONNECTION_STRING: z.string(),
+  HOST: z.string().default('0.0.0.0'),
+  DB_CONNECTION_STRING: z.string().default('postgres://postgres:mysecretpassword@localhost:5432/dialogporten'),
   APPLICATIONINSIGHTS_CONNECTION_STRING: z.string().optional(),
+  PORT: z.coerce.number().default(3000),
+  MODE: z.enum(['development', 'production']).default('development'),
+  DEV_ENV: z.string().default('dev'),
+  OIDC_URL: z.string().default('test.idporten.no'),
+  HOSTNAME: z.string().default('http://localhost'),
+  ENABLE_APP_INSIGHTS: z.string().default('false'),
+  REFRESH_TOKEN_EXPIRES_IN: z.coerce.number().default(3600),
+  CLIENT_ID: z.string(),
+  CLIENT_SECRET: z.string(),
+  COOKIE_NAME: z.string(),
+  SESSION_SECRET: z.string().min(32).default('SecretHereSecretHereSecretHereSecretHereSecretHereSecretHereSecretHere'),
+  ENABLE_HTTPS: z.boolean().default(false),
+  COOKIE_MAX_AGE: z.coerce.number().default(30 * 24 * 60 * 60 * 1000),
 });
 
 const env = envVariables.parse(process.env);
 
-const port = process.env.PORT || 3000;
-const isAppInsightsEnabled = process.env.ENABLE_APP_INSIGHTS === 'true';
-const isDev = process.env.DEV_ENV === 'dev';
-
-export default {
+const config = {
   version: env.GIT_SHA,
-  port,
-  isAppInsightsEnabled,
-  isDev,
+  port: env.PORT,
+  isAppInsightsEnabled: env.ENABLE_APP_INSIGHTS === 'true',
```

Comment:
Mulig personal preference her altså, men er det litt bedre å strukturere disse under egne keys? Vi har en applicationInsights: {} her allerede vi kunne satt denne under. 

```typescript
{
   applicationInsights: {
        enabled: env.ENABLE_APP_INSIGHTS,
        connectionString: env.APPLICATIONINSIGHTS_CONNECTION_STRING
        ....
   }
}
```


client_id og client_secret f.eks under `openID`eller hva de brukes til

Reply:
Er vel IDPorten ser jeg nå ja. Gjelder vel også hostname, oidc_url med mer

Reply:
Ikke uenig, bare scope bomb for denne PR-en. Dvs vi er ikke helt i mål med konfigurasjonen nå, har zoomet altfor mye inn på Fastify-delen av det i denne PR-en til å rettferdiggjøre strafferunder med konfig.

Reply:
Enig, men er flisespikkeri som eventuelt kan fikses i egen PR.